packages/mrm-core/src/npm.js

```diff
@@ -265,6 +266,14 @@ function getUnsatisfiedDeps(deps, versions, options) {
 	});
 }
 
+function config() {
```

Comment:
I'm now thinking: should it be part of the `package` task instead. It's no entirely generic anyway since it only returns two fields.

Reply:
Sure. It had looked like these were the only two fields related to `package.json` which your user meta package wasn't covering. But sure, I can remove it from core and just leave it in the `package` task.

As far as the unrelated changes, I built this on top of my testing additions to more easily ensure the changes of this PR weren't regressing on coverage. But I can rebase the changes directly back on `master`.

Also, thanks for the project invite! Let me know your approach for commits; do you want all commits to go through PRs with the access allowing me to just handle approved merges, or do you want me to directly add minor fixes, tests, docs, or refactoring without PRs?

(Heading out now)

Reply:
Better with pull requests but now you don't have to maintain a fork ;-)